fdbserver/RestoreApplier.actor.cpp

```diff
@@ -159,14 +158,10 @@ ACTOR static Future<Void> handleSendMutationVectorRequest(RestoreSendVersionedMu
 				}
 			}
 			// Note: Log and range mutations may be delivered out of order. Can we handle it?
-			if (mutation.type == MutationRef::SetVersionstampedKey ||
```

Comment:
Can we add an ASSERT_WE_THINK() here to ensure there is no VersionStampedKey in Applier?
I remembered we may have this assert somewhere else. I still think the extra assert is just to help catch any surprise and pin down the problem quickly.